radix-engine-interface/src/model/auth/access_rules/invocations.rs

```diff
@@ -33,7 +33,7 @@ impl Into<NativeFnInvocation> for AccessRulesAddAccessCheckInvocation {
 pub struct AccessRulesSetAccessRuleInvocation {
     pub receiver: RENodeId,
     pub index: u32,
-    pub key: AccessRuleKey,
+    pub selector: AccessRuleSelector,
```

Comment:
Not sure if we gain anything by using a selector here over creating a new set of invocations for default?

Reply:
Josh and I discussed this during a call. We have agreed to remove the `AccessRuleSelector` and have a set of invocations for methods and another set for groups. This has been addressed in [23bdce8](https://github.com/radixdlt/radixdlt-scrypto/pull/624/commits/23bdce82262394d2f47a952b2d0906ef9fb1fb42)